src/app/theme/panel.css

```diff
@@ -35,11 +35,11 @@
 }
 
 .ck.ck-balloon-panel [aria-label="Mermaid toolbar"] .ck-button {
-	line-height: 1;
+	line-height: 21px;
```

Comment:
If I'm not mistaken this is exactly the same value that is assigned in generic styles for `.ck-button`.

If that's the case there's no reason to explicitly override it for the same value - just remove the rule. Less is better.